backend/tests/Feature/UserQueryTest.php

```diff
@@ -0,0 +1,122 @@
+                        'linkedin' => 'regularuser',
+                        'academia_edu_id' => 'regularuser',
+                        'skype' => 'regularuser',
+                        'zoom' => 'regularuser',
+                    ],
+                    'professional_title' => 'Regular User',
+                    'specialization' => 'Regular',
+                    'affiliation' => 'Regular Users',
+                    'interest_keywords' => [
+                        'regular',
+                        'user',
+                    ],
+                    'disinterest_keywords' => [
+                        'nonregular',
+                        'irregular',
+                    ],
+                    'biography' => 'I am a regular user.',
+                    'profile_picture' => 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII=',
+                    'websites' => [
+                        'https://github.com',
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider profileMetadataProvider
+     * @return void
+     */
+    public function testThatUserDetailsCanBeQueried(?array $profile_metadata): void
+    {
+        $user = User::factory()->create([
+            'name' => 'Regular User',
+            'email' => 'regularuser@gmail.com',
+            'username' => 'regularuser',
+            'profile_metadata' => $profile_metadata,
+        ]);
+        $response = $this->graphQL(
+            'query getUser ($id: ID!) {
+                user (id: $id) {
+                    profile_metadata {
+                        salutation
+                        phone
+                        mailing_address {
+                            city
+                            state
+                            postal_code
+                            street_address
+                        }
+                        orchid_id
+                        humanities_commons
+                        social_media {
+                            google
+                            twitter
+                            facebook
+                            instagram
+                            linkedin
+                            academia_edu_id
+                            skype
+                            zoom
+                        }
+                        professional_title
+                        specialization
+                        affiliation
+                        interest_keywords
+                        disinterest_keywords
+                        biography
+                        profile_picture
+                        websites
+                    }
+                }
+            }',
+            [ 'id' => $user->id ]
+        );
+        $response->assertJson([
+            'data' => [
+                'user' => [
+                    'profile_metadata' => $profile_metadata,
+                ],
+            ],
+        ]);
```

Comment:
I tweaked this to be a little less repetitive in this case.  Feel free to revert if you have a strong opinion that I should remain the way it was.

Reply:
I do like your simplification. I restored the assertions for the basic user data, however, because I don't think we should overlook these.